Assets/Tests/InputSystem/CoreTests_Editor.cs

```diff
@@ -545,6 +545,31 @@ public void Editor_InputAsset_CanReplaceBindingGroupThroughSerialization()
         Assert.That(map.bindings[0].groups, Is.EqualTo(""));
     }
 
+    [Test]
+    [Category("Editor")]
+    public void Editor_InputActionAssetManager_SaveChangesToAsset_DoesNotThrow_WhenParentDirectoryWasRenamed()
```

Comment:
`Editor_InputActionAssetManager_CanMoveAssetOnDisk`? The throwing seems more incidental to the previous bug than instrumental to the behavior of interest here.